client/src/app/dashboard/datatable-notification/datatable-notification.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, Inject, OnInit, ViewChild} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialog, MatPaginator, MatSort} from '@angular/material';
+import { DatatableNotificationDataSource } from './datatable-notification-datasource';
+
+export interface DialogData {
+  animal: 'panda' | 'unicorn' | 'lion';
```

Comment:
I assume this is related to the missing dialog call? Currently, this interface is not used in any component so if it is not needed anymore please remove it. If you use it in the currently missing openDialog() method please make sure to alter it to be a realistic data model for our application.